src/components/Calculator.js

```diff
@@ -1,6 +1,39 @@
 import PropTypes from 'prop-types';
+import { useState } from 'react';
+import calculate from '../logic/calculate';
 
 function Calculator() {
+  const [calculator, setCalculator] = useState(
+    {
+      total: null,
+      next: null,
+      operation: null,
+    },
+  );
+
+  const [displayNumber, setDisplayNumber] = useState('0');
+
+  const byPassNext = (calculator) => {
+    if (calculator.next) {
+      return setDisplayNumber(calculator.next);
+    }
+    if (calculator.total) {
+      return setDisplayNumber(calculator.total);
+    }
+    return setDisplayNumber('0');
+  };
+  const clickCalc = (value) => {
+    const newCalculator = calculate(calculator, value);
+    setCalculator(newCalculator);
+    byPassNext(newCalculator);
+  };
+
+  /*
+  const clickOperation = (value) => {
+    setCurrentNumber(value);
+  };
+  */
```

Comment:
- [ ] For codes that are not meaningful to our project and does not perform anything in the project so far, it's best to remove them and not comment on them. It's an excellent convention not to have commented codes in our codebase as it makes your code clean, lean, and professional. Kindly remove all commented codes from your codebase 👍🏽